shared/chat/inbox/index.native.tsx

```diff
@@ -251,17 +249,19 @@ class Inbox extends React.PureComponent<T.Props, State> {
             </Kb.Box2>
           ) : (
             <FlashList
-              estimatedItemSize={64}
-              overScrollMode="never"
+              disableAutoLayout={true}
```

Comment:
there are some issues w/ flashlist when the item sizes are fixed, it doesn't trigger the subview layout as it expects so it'll lay things out incorrectly sometimes. Since this is all fixed size we can just skip that whole part